.github/workflows/lint-and-test.yml

```diff
@@ -23,17 +30,13 @@ jobs:
       - name: Install All Dependencies
         run: yarn
 
-      - name: Check Formatting with Prettier
-        run: yarn format-check-all
-
-      - name: Lint JS with ESLint
-        run: yarn lint
-
-      - name: Lint CSS with Stylelint
-        run: yarn lint-css
-
-      - name: Check TypeScript compilation
-        run: yarn lint-ts
+      - name: Lint and fix all changed files
+        run: yarn lint-staged -q --no-stash --diff "origin/main...HEAD"
 
-      - name: Run tests
-        run: yarn test
+      - name: Commit and push changes
+        id: commit
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add -A
+          git commit -m "Fix unlinted files" --no-verify && git push || true
```

Comment:
Can we be sure with `--no-verify` here? I wonder if it could be a future unknown place that can introduce errors.

Reply:
Without --no-verify there's an infinite loop, since the pre-commit pipeline is just lint-staged and it's run explicitly it should be okay. Your concern is valid, though.